src/applications/check-in/utils/api/index.js

```diff
@@ -1,6 +1,11 @@
 import { createApiEvent } from '../analytics';
 import recordEvent from 'platform/monitoring/record-event';
 
+/**
+ * @param {Promise} request
+ * @param {string} [eventName]
+ * @param {string} [token]
+ */
```

Comment:
Are you only specifying the return type where the type checker can't infer it? If so maybe we should also specify that in the readme so we can do things consistently 🙂

Reply:
Thanks @acrollet! You are absolutely correct about the return types and I'll update readme. The type-checker in the editor is really good at inferring types for code that is self-contained, so I only included types where types could not be identified without them.

Reply:
sweet!